service/src/main/java/bio/terra/workspace/service/crl/CrlService.java

```diff
@@ -379,4 +385,32 @@ public AzureResourceManager getResourceManager(AzureCloudContext azureCloudConte
         .withTenantId(azureCloudContext.getAzureTenantId())
         .withSubscription(azureCloudContext.getAzureSubscriptionId());
   }
+
+  private ComputeManagerCow buildComputeManagerCow(
+      AzureCloudContext azureCloudContext, AzureConfiguration azureConfig) {
+    TokenCredential azureCreds = getManagedAppCredentials(azureConfig);
+
+    AzureProfile azureProfile =
+        new AzureProfile(
+            azureCloudContext.getAzureTenantId(),
+            azureCloudContext.getAzureSubscriptionId(),
+            AzureEnvironment.AZURE);
+    return ComputeManagerCow.create(clientConfig, azureCreds, azureProfile);
```

Comment:
This might be slightly different in the latest CRL code, gonna pull this down and try it too..